routes/routes.go

```diff
@@ -1,77 +1,71 @@
 package routes
 
 import (
+	"log"
 	"net/http"
 
 	"github.com/arshamalh/blogo/controllers"
 	"github.com/arshamalh/blogo/databases"
-	"github.com/arshamalh/blogo/middlewares"
 	"go.uber.org/zap"
 
-	"github.com/arshamalh/blogo/models/permissions"
 	"github.com/labstack/echo/v4"
 	emdware "github.com/labstack/echo/v4/middleware"
 )
 
 func InitializeRoutes(router *echo.Echo, db databases.Database, logger *zap.Logger) {
+
 	// Basic configurations and middleware.
 	router.Use(emdware.CORSWithConfig(emdware.CORSConfig{
 		AllowOrigins: []string{"http://localhost:3000"},
 	}))
 
-	router.Use(middlewares.ZapLogger(logger))
-
 	router.GET("/health", func(c echo.Context) error {
+		log.Printf("Health check request received")
 		return c.JSON(http.StatusNoContent, nil)
 	})
 
-	/// ** Routes
-
+	// Routes for users
 	user_routes := router.Group("/api/v1/users")
 	uc := controllers.NewUserController(db, logger)
 	{
-		user_routes.POST("/register", uc.UserRegister)
-		user_routes.POST("/login", uc.UserLogin)
-		user_routes.POST("/logout", uc.UserLogout, middlewares.RequireLogin)
-		user_routes.GET("/check_username", uc.CheckUsername)
-		user_routes.GET("/id", uc.UserID, middlewares.RequireLogin)
-		// Get & Update Profile and more
-		// user_routes.GET("/info/:id", controllers.UserInfo)
-	}
+		user_routes.POST("/register", func(c echo.Context) error {
+			log.Printf("User registration request received")
```

Comment:
Logger middleware is doing exactly what you are doing here, so don't change these handlers.